### PR TITLE
Add Socrata API

### PR DESCRIPTION
**Pull request by nair0lf32**
Add Socrata API to Open Data

- [x] My submission is formatted according to the guidelines in the [contributing guide](/CONTRIBUTING.md)
- [x] My addition is ordered alphabetically
- [x] My submission has a useful description
- [x] The description does not end with punctuation
- [x] Each table column is padded with one space on either side
- [x] I have searched the repository for any relevant issues or pull requests
- [x] Any category I am creating has the minimum requirement of 3 items
- [x] All changes have been [squashed][squash-link] into a single commit

[squash-link]: <https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -791,6 +791,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Quandl](https://www.quandl.com/) | Stock Market Data | No | Yes | Unknown |
 | [Recreation Information Database](https://ridb.recreation.gov/) | Recreational areas, federal lands, historic sites, museums, and other attractions/resources(US) | `apiKey` | Yes | Unknown |
 | [Scoop.it](http://www.scoop.it/dev) | Content Curation Service | `apiKey` | No | Unknown |
+| [Socrata](https://dev.socrata.com/) | Access to Open Data from Governments,Non-profits and NGOs around the world | `OAuth` | Yes | Yes |
```
Comment (matheusfelipeog) [inline README.md:794]: Fix this, please:

```suggestion
| [Socrata](https://dev.socrata.com/) | Access to Open Data from Governments, Non-profits and NGOs around the world | `OAuth` | Yes | Yes |
```

**Review by matheusfelipeog — CHANGES_REQUESTED** [review]
Thanks for your contribution @nair0lf32 😆 

Can you resolve this review, please?

**Comment by nair0lf32** [comment]
Done. Thank you for the suggestion.

**Review by matheusfelipeog — APPROVED** [review]
Now it's Ok. a big THANKS for your update ;)

**Pull request merged**